src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -25,10 +25,26 @@ function logCapturedError(capturedError : CapturedError) : void {
       willRetry,
     } = capturedError;
 
+    const {
+      message,
+      name,
+      stack,
+    } = error;
+
+    const errorSummary = message
+      ? `${name}: ${message}`
+      : name;
+
     const componentNameMessage = componentName
       ? `React caught an error thrown by ${componentName}.`
       : 'React caught an error thrown by one of your components.';
 
+    // Error stack varies by browser; normalize it for our logging.
+    const formattedCallStack = stack.replace(new RegExp(`^${errorSummary}\n`), '')
```

Comment:
Thanks for pointing that out Ben.

The latest commit has been tested with single-line message, multi-line message, no message, and messages containing special regex chars.

Browsers tested: Chrome, Firefox, Safari, IE